docs/SETTING_UP_A_NEW_INSTANCE.md

```diff
@@ -10,21 +10,39 @@ DCAF has a lot of the surrounding infrastructure set up in such a way that it's
 
 ### Create a new heroku app
 
-[Click this here link to spin up a new app](https://heroku.com/deploy?template=https://github.com/DCAFEngineering/dcaf_case_management). Follow the directions to fill out environment variables and configuration. (Ask our slack channel if you have questions.)
+[Click this here link to spin up a new app](https://heroku.com/deploy?template=https://github.com/DCAFEngineering/dcaf_case_management). By deploying using this link, the following accounts will be automatically provisioned and managed via Heroku (as heroku add-ons). This means that each individual app deployed with this link will have its own unique accounts with the following:
+
+* mLabs -- this is a mongodb data service
+* Sendgrid -- this is used as the email service
+* Logentries -- used for application logging
+* Scheduler -- used to run tasks on a cron schedule
+
+*Note -- if you choose to deploy this app manually rather than via the link provided above, you will need to handle your own set up for your mongodb, email, logging, and cron services.*
+
+Follow the directions to fill out necessary environment variables and configuration for your DARIA instance. (Ask our slack channel if you have questions.)
 
 * Go into logentries and add anyone who needs to be added to the alerts. This should be just people who need to know about application errors, pretty much.
-* Set up a google cloud account and generate OAuth credentials (Google Cloud Platform -> API Manager -> Credentials -> Create Credentials, can be set up alongside existing creds for different environments)
 * Set up an uptime monitor (we recommend StatusCake!)
 * Click on `Scheduler` and set up the scheduled cleanup job:
 - `$ rake nightly cleanup`, dyno size hobby, frequency daily, 08:00 UTC
-
-Optionally but recommended:
-
-## Optional steps
-
-* Set `SKYLIGHT_AUTH_TOKEN` (get it from Skylight, this is a performance monitoring service)
-* Set `SQREEN_TOKEN` (get it from Sqreen, this is a security service)
-* if you have a custom domain set up, set up heroku ACM
+* Set up Google OAuth credentials
+  * Set up a google account to administer this app (or choose an existing account you own)
+  * Once logged in, navigate to https://console.developers.google.com/apis/dashboard
+  * Add a project (note -- credentials can be set up alongside existing creds for different environments)
+  * Enable the Google+ API (this is necessary to use google for sign in to the app)
+  * Click to "Create OAuth client ID"
```

Comment:
I think these are configurations of an oauth client id, right? 

```suggestion
  * Click to "Create OAuth client ID"
    * Set your `Authorized Javascript Origins` to: https://your-app.herokuapp.com 
    * Set your `Authorized redirect URIs` to: https://your-app.herokuapp.com/users/auth/google_oauth2/callback
```

Reply:
Yes... is OAuth client not what's needed for the google sign in? Or are you saying you'd rather not go into that level of detail?

Reply:
Oh, no, it's definitely needed! But I think setting these values is tied to creating the oauth client id, not configuring the consent screen. I might be wrong about that, let me check.

Reply:
yep, it looks like the consent screen is some different stuff, and the authorized origins and redirects are on the client id/client secret level:

Here's the oauth screen:

![image](https://user-images.githubusercontent.com/3866868/52172850-f4838f00-2745-11e9-809a-b77da5ad720e.png)

Also, this made me look at authorized javascript origins for the existing client ids we have set up for oauth, and none of them have authorized javascript origins, so we might be able to remove that line of setup. Do the keys still work if you empty out that line in your config?